prov/rxm/src/rxm_cq.c

```diff
@@ -137,6 +137,24 @@ static int rxm_match_rma_iov(struct rxm_recv_entry *recv_entry,
 	return FI_SUCCESS;
 }
 
+static int rxm_finish_buf_recv(struct rxm_rx_buf *rx_buf)
+{
+	uint64_t flags = rx_buf->pkt.hdr.flags | FI_RECV;
+
+	if (rx_buf->pkt.hdr.op == ofi_op_msg) {
+		flags |= FI_MSG;
+	} else {
+		assert(rx_buf->pkt.hdr.op == ofi_op_tagged);
+		flags |= FI_TAGGED;
+	}
```

Comment:
Need to set FI_MORE based on size.

Reply:
I want to defer adding this until open PRs which touch the same parts of the code get merged. I can add a TODO and an assert. Muti-rail is the only consumer of FI_BUFFERED_RECV mode for now.

Reply:
sounds good

Reply:
Thanks. I've updated the PR now.